modules/distribution/product/src/main/assembly/bin.xml

```diff
@@ -1584,16 +1568,16 @@
             <outputDirectory>wso2am-${pom.version}/repository/resources/security/</outputDirectory>
         </file>
         <!-- Coping lerna related files -->
-        <file>
-            <source>src/main/conf/lerna/lerna.json</source>
-            <outputDirectory>wso2am-${pom.version}/repository/deployment/server/jaggeryapps</outputDirectory>
-            <destName>lerna.json</destName>
-        </file>
-        <file>
-            <source>src/main/conf/lerna/package.json</source>
-            <outputDirectory>wso2am-${pom.version}/repository/deployment/server/jaggeryapps</outputDirectory>
-            <destName>package.json</destName>
-        </file>
+<!--        <file>-->
```

Comment:
Should we remove this completely instead of commenting out?

Reply:
+1